demos/python_demos/3d_segmentation_demo/README.md

```diff
@@ -49,6 +49,12 @@ Options:
   -c PATH_TO_CLDNN_CONFIG, --path_to_cldnn_config PATH_TO_CLDNN_CONFIG
                         Required for GPU custom kernels. Absolute path to an
                         .xml file with the kernels description.
+  -ms N1,N2,N3,N4, --mri_sequence N1,N2,N3,N4
+                        Optional. Set MRI-sequence, if data is in single NIFTI
+                        file. Input order is: native T1, native T2, T2-FLAIR,
+                        post-Gadolinium contrast T1
+  --full_intensities_range
```

Comment:
Please, `--full_intensities_range` should be optional. `Optional. Take intensities of the input image in a full range.`

Reply:
Ok